cli.go

```diff
@@ -1468,7 +1468,18 @@ func GithubListReposByMetaSearch(query string) ([]*github.Repository, error) {
 	return ghClient.SearchRepos(query)
 }
 func GithubListReposByCodeSearch(query string) ([]*github.Repository, error) {
-	return ghClient.SearchCode(query)
+	var repos []*github.Repository
+
+	codeResults, err := ghClient.SearchCode(query)
+	if err != nil {
+		return nil, err
+	}
+
+	for _, codeResult := range codeResults {
+		repos = append(repos, codeResult.Repository)
```

Comment:
You can directly check whether it's a fork here, and get `codeResult.Repository.GetHTMLURL()`.

Those are the two things we actually need the repository object.

Reply:
No need to compile another list :)

Reply:
Them you can use the `Deduplicate` function to remove duplicate urls.

Reply:
Yeah I meant that we didn't need the actual `CodeResults` but the `Repository` itself. Nvm, I think it works fine now ^^.

Reply:
> Those are the two things we actually need the repository object.

Doesn't the function calling that function already do that?

> Them you can use the `Deduplicate` function to remove duplicate urls.

And this too :P

Reply:
I'll give you better comments when I'm not replying from my smartphone 😅

But yeah, you got the right idea

Reply:
Hahah no worries, there's no need to rush. Thanks ^^